inc/Engine/Media/AboveTheFold/WarmUp/Controller.php

```diff
@@ -237,9 +238,15 @@ public function add_wpr_imagedimensions_query_arg( string $url ): string {
 	/**
 	 * Check if the current request is for mobile.
```

Comment:
Not related to the PR but we could update this docstring: it looks like a bad copy-paste. What this is checking is not related to a current request, it's just checking if the current config does separate the mobile cache.